sdcm/sct_runner.py

```diff
@@ -50,14 +51,6 @@ def instance_type(test_duration):
             return "r5.large"
         return "t3.large"  # has 7h 12m CPU burst
 
-    @staticmethod
-    def instance_root_disk_size(test_duration):
-        disk_size = 35  # GB
-        if test_duration and test_duration > 3 * 24 * 60:  # 3 days
-            # add 20G more space for jobs which test_duration is longer than 3 days
```

Comment:
Can we still use 35G for job is less than 1 day?

Reply:
changed to 80GB base size.

Reply:
changed to 80GB base size.